lib/internal/Magento/Framework/View/Test/Unit/Asset/Bundle/ManagerTest.php

```diff
@@ -118,6 +118,8 @@ public function testAddAssetWithExcludedFile()
         $this->asset->expects($this->atLeastOnce())
             ->method('getFilePath')
             ->willReturn('source/file.min.js');
+        $this->asset->method('getPath')
+            ->willReturn('');   // PHP 8.1. compatibility
```

Comment:
please use something similar to "real" value in future

Reply:
Okay:)